cli/client_test.go

```diff
@@ -37,15 +37,26 @@ func (tw *testWriter) Write(b []byte) (int, error) {
 // setup creates a new cli.Context and viamClient with fake auth and the passed
 // in AppServiceClient and DataServiceClient. It also returns testWriters that capture Stdout and
 // Stdin.
-func setup(asc apppb.AppServiceClient, dataClient datapb.DataServiceClient) (*cli.Context, *viamClient, *testWriter, *testWriter) {
+func setup(asc apppb.AppServiceClient, dataClient datapb.DataServiceClient,
+	defaultFlags *map[string]string,
+) (*cli.Context, *viamClient, *testWriter, *testWriter) {
 	out := &testWriter{}
 	errOut := &testWriter{}
 	flags := &flag.FlagSet{}
+	// init all the default flags from the input
+
+	if defaultFlags != nil {
```

Comment:
@benjirewis I only added you based on this convo for default flags :D let me know if this is what you were thinking too. Feel free to ignore the rest of the PR.

Reply:
Yeah this looks good to me.